src/azul/service/hca_response_v5.py

```diff
@@ -465,7 +487,12 @@ def choose_entry(_term):
             term_object_params = {'term': choose_entry(term), 'count': term['doc_count']}
             if 'myProjectIds' in term:
                 term_object_params['projectId'] = [bucket['key'] for bucket in term['myProjectIds']['buckets']]
-            term_list.append(TermObj(**term_object_params))
+            if contents['myTerms']['meta']['path'][-1] == 'organism_age':
+                t = term_object_params.get('term')
```

Comment:
adress this https://github.com/DataBiosphere/azul/pull/1909/files#r453754209

Reply:
The `.get` has two purposes here. One, ensure that `term` is present in the `term_object_params ` dict. Two, catches if the actual content of `term` is None. However, we know that `term` is always present in this dict, given that it gets declared above, in line [454](https://github.com/DataBiosphere/azul/pull/1909/files#diff-bce0a0b2167abef4edf00341594e947bR454). We can still use the try-catch block on `t.partition(' ')` since it might attempt to partition a None. Let me know what you think.

Reply:
In that case, 

```suggestion
                t = term_object_params['term']
```